### PR TITLE
Japanese Translation

### PR DESCRIPTION
**Pull request by pklion**
I made a Japanese translation of index.html.
(I didn't make the menu to change language, sorry! :) )

### CODE REVIEW

File: index_ja.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (5E7EN) [inline index_ja.html:2]: `<html lang="ja">`

**Review by 5E7EN — CHANGES_REQUESTED** [review]
Seems fine, although since I'm not fluent in Japanese I can't exactly tell for sure, but I'll merge this anyway once the `lang` attribute is changed. Perhaps you'd also like to add some link in the English version to give users an opportunity to switch languages?

File: index_ja.html (hunk 1 above)
Comment (pklion) [reply index_ja.html]: Thank you for pointing it out, and I fixed the lang attribute and some meta attribute too.
I'll make a menu to switch language, so please wait.

**Comment by pklion** [comment]
image
Done!

**Review by 5E7EN — APPROVED** [review]

**Comment by pklion** [comment]
Thank you!

**Pull request merged**